granadilla/cli.py

```diff
@@ -123,6 +123,49 @@ def _get_good_password(self, user):
         self.success("Password is strong enough (bruteforce: %s)", check['crack_time_display'])
         return password1
 
+    def change_device_password(self, device):
+        password = None
+
+        while password is None:
+            password = self._get_good_device_password(device)
+
+        device.set_password(password)
+
+    def _get_good_device_password(self, device):
+        password1 = self.grab("Password: ", True)
+        password2 = self.grab("Password (again): ", True)
+        if password2 != password1:
+            self.error("Passwords do not match, try again.")
+            return None
+
+        blacklist = [
+            device.device_owner,
+            device.device_type,
+        ]
```

Comment:
If the only difference with the ``_get_good_password()`` method is the handling of the blacklist, we should change the ``_get_good_password()`` method so that it takes a ``blacklist`` instead of a ``user``, and reuse it here instead.

What do you think?